### PR TITLE
Define the structure related to ECDAA

### PR DESCRIPTION
**Pull request by akakou**
Add the structure related to ECDAA. 
Please read https://github.com/google/go-tpm/pull/285#issuecomment-1170212446 .

- [x] TPMS_SCHEME_ECDAA added 
- [x] A type alias for internal.TPMSSchemeECDAA as tpms.SchemeECDAA

### CODE REVIEW

File: direct/structures/internal/structures.go [hunk 1]
```diff
@@ -1044,8 +1044,18 @@ type TPMSSchemeHash struct {
 	HashAlg TPMIAlgHash
 }
 
+// TPM represents a TPMS_SCHEME_ECDAA.
```
Comment (chrisfenner) [inline direct/structures/internal/structures.go:1047]: nit: "TPMSSchemeECDAA represents a ..."

All Go documentation comments are complete sentences beginning with the thing being documented - https://go.dev/blog/godoc

Comment (akakou) [reply direct/structures/internal/structures.go]: @chrisfenner 

Oops...It was my mistake, thanks.
And I fixed it.

**Comment by akakou** [comment]
@chrisfenner 

I fixed the mistake, so could you confirm it?

**Comment by chrisfenner** [comment]
Looks great, thanks @akakou! Sorry for the delay. We had 2 company-wide days off due to the US holiday this weekend.

**Comment by akakou** [comment]
@chrisfenner 
Thank you!

**Pull request merged**